output/mistral_influxdb/mistral_influxdb.c

```diff
@@ -802,8 +809,20 @@ void mistral_received_data_end(uint64_t block_num, bool block_error)
         int *tag_p = tag_set;
         while (*tag_p != FIELD_ID_MAX) {
             failed |= (putc(',', post_data) < 0);
-            failed |= (fprintf(post_data, "%s=\"%s\"", fields[*tag_p].name,
-                               fields[*tag_p].value) < 0);
+            switch (*tag_p) {
+                /* InfluxDB doesn't seem to take quotation marks with tag values EXCEPT for tags
+                 * representing file paths. */
```

Comment:
Is this definitely true. It may just have been the value being passed as an integer rather than a float that Influx was complaining about.

Reply:
file paths should definitely use quotations in case there is a space in them, though i dont believe the integer/float change to be a problem at all since the type of each field is user defined and not something expected by influxdb. going to have to hold off testing until the bin labels are fixed.

Reply:
following up: while it doesnt matter for *new* databases, we will want to make value still be passed in as a float for compatibility with *existing* databases as otherwise influx will return back with HTTP 400